src/modlunky2/levels/tile_codes.py

```diff
@@ -261,7 +366,7 @@ def get(self, name):
         return self._inner.get(name)
 
     def set_obj(self, obj: "TileCode"):
-        obj.validate()
+        # obj.validate()
```

Comment:
this should be fine to keep, just remove `validate_name` from `validate` so we continue to validate values

Reply:
Think I fixed both issues the way they should be